LayoutTests/inspector/console/js-completions.html

```diff
@@ -52,6 +52,27 @@
         }
     });
 
+    suite.addTestCase({
+        name: "console.jsCompletions.completionOrdering",
+        description: "Test that the JavaScript completions are ordered correctly.",
+        async test() {
+            await InspectorTest.evaluateInPage(`myObject = {
+                1: 42, 2: 42, 10: 42, 20: 42, aa: 42, ab: 42, Aa: 42, Ab: 42, __aa__: 42, __ab__: 42
+            };`);
+            WI.javaScriptRuntimeCompletionProvider.clearCachedPropertyNames();
+
+            InspectorTest.json(await generateJSCompletions({baseString: "myObject.", prefix: ""}));
```

Comment:
NIT: it would be nice to maybe at least log something like "Generating completions for `myObject.`..." before each of these, as that way there's at least some context in the output file (i.e. if this test becomes flaky then it's potentially possible to just look at the output to see what went wrong instead of also having to look at this file too)